app/models/category.rb

```diff
@@ -4,6 +4,8 @@ class Category < ApplicationRecord
   has_many :users, through: :runs
   has_many :rivalries, dependent: :destroy
 
+  has_one :srdc, class_name: 'SpeedrunDotComCategory', dependent: :destroy
```

Comment:
`srdc` is used around the app to indicate speedrun.com by itself, should this be something like `srdcc` to indicate it's not just a reference to SRDC the site?

Reply:
It's not obvious what `srdcc` (or `srdcg`, or `srdcu`) stands for, so if we did I'd want to do something more like `srdc_category`. But imo the benefit doesn't outweigh the unwieldiness, because the context reveals the meaning, e.g.

```ruby
@game.category.srdc # a Speedrun.com category
@game.srdc          # a Speedrun.com game
@user.srdc          # a Speedrun.com user
current_user.srdc   # a Speedrun.com user
```

We'd never be faced with an object that has two Speedrun.com connections -- a `Category` only relates to a Speedrun.com category and a `Game` only relates to a Speedrun.com game, etc. If that turns out to be wrong and we eventually do have two connections in one object, we can update the relation names then.

We do this for other connections too:
```ruby
current_user.twitch
current_user.patreon
current_user.google
```

The one place context isn't as obvious is in instance methods, like inside `Category#sync_with_srdc!` where you find `srdc` naked because `self.` is implied. So far I think that's still not a huge enough deal to increase the name lengths, but I'm open to argument.